policyengine_us/parameters/gov/states/co/tax/income/subtractions/able_contribution/cap.yaml

```
@@ -6,19 +6,25 @@ metadata:
   breakdown:
     - filing_status
   reference:
-    - title: 2023 Colorado Individual Tax Filing Guide 104 Book 
+    - title: 2023 Colorado Individual Tax Filing Guide 104 Book - Subtractions Line 11
       href: https://tax.colorado.gov/sites/tax/files/documents/Book0104_2023.pdf#page=14
+    - title: 2024 Colorado Individual Tax Filing Guide 104 Book - Subtractions Line 11
+      href: https://tax.colorado.gov/sites/tax/files/documents/DR0104_book_2024.pdf#page=20
```

Page with the actual values
```suggestion
      href: https://tax.colorado.gov/sites/tax/files/documents/DR0104_book_2024.pdf#page=21
```